src/main/java/ru/svgogin/service/spark/errordto/CompanyErrorDto.java

```diff
@@ -0,0 +1,31 @@
+package ru.svgogin.service.spark.errordto;
+
+import org.springframework.lang.NonNull;
+
+public final class CompanyErrorDto {
```

Comment:
```suggestion
public final class ErrorDto {
```
This class doesn't depend on company

Reply:
Done